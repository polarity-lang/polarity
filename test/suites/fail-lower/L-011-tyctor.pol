data Bool { True, False }

data Option(elem: Type) {
    None(elem: Type): Option(elem),
    Some(elem: Type, x: elem): Option(elem),
}

let example2: Option(wrong := Bool) { None(Bool) }
