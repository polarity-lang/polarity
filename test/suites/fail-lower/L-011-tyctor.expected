L-011

  × Mismatched named arguments: given wrong, expected elem
   ╭─[L-011-tyctor.pol:7:1]
 7 │ 
 8 │ let example2: Option(wrong := Bool) { None(Bool) }
   ·                               ────
   ╰────
